tests/route/test_route_perf.py

```diff
@@ -0,0 +1,114 @@
+import pytest
+import json
+from datetime import datetime
+from tests.common.utilities import wait_until
+
+pytestmark = [
+    pytest.mark.topology('any'),
+    pytest.mark.device_type('vs')
+]
+
+ROUTE_TABLE_NAME = 'ASIC_STATE:SAI_OBJECT_TYPE_ROUTE_ENTRY'
+
+def prepare_dut(duthost):
+    # Set up interface
+    duthost.shell('sudo config interface ip add Ethernet72 10.1.0.54/31')
```

Comment:
Not every platforms have `Ethernet72`. Is this intended to run only on some platform?

Reply:
Updated the interface used for the test. Also added IPv6 and multiple neighbors.